contracts/interfaces/ICompliance.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity 0.5.16;
+
+import "./IContractRegistry.sol";
+
+/// @title Elections contract interface
+interface ICompliance is Ownable { 
+	event ValidatorComformenceUpdare(address validator, string type);
+
+	/*
+     * External methods
+     */
+
+    /// @dev Called by a validator as part of the automatic vote unready flow
```

Comment:
bad c&p? :) (also in rest of the functions)

Reply:
fixed